src/EditorFeatures/CSharp/UseAutoProperty/UseAutoPropertyAnalyzer.cs

```diff
@@ -81,11 +83,16 @@ private bool CheckExpressionSyntactically(ExpressionSyntax expression)
         protected override ExpressionSyntax GetGetterExpression(IMethodSymbol getMethod, CancellationToken cancellationToken)
         {
             var getAccessor = getMethod.DeclaringSyntaxReferences[0].GetSyntax(cancellationToken) as AccessorDeclarationSyntax;
-            var firstStatement = getAccessor?.Body?.Statements.SingleOrDefault();
-            if (firstStatement?.Kind() == SyntaxKind.ReturnStatement)
+            var statements = getAccessor?.Body?.Statements ?? SpecializedCollections.EmptyEnumerable<StatementSyntax>();
```

Comment:
`SingleOrDefault()` throws if there are 2 or more items.  In fact, without my changes to fail the test on any analyzer exceptions, the test `TestGetterWithMultipleStatements` would throw, but that exception was turned into an analyzer diagnostic that was ignored and the test erroneously passed.